compiler/src/main/java/com/squareup/anvil/compiler/codegen/reference/AnnotationReferenceIr.kt

```diff
@@ -37,6 +40,17 @@ internal class AnnotationReferenceIr(
       message = "The declaring class was null, this means the annotation wasn't used on a class."
     )
 
+  val parentScope: ClassReferenceIr
+    get() = context.referenceClass(annotation.parentScope())?.toClassReference(context)
```

Comment:
Regarding our Slack conversation: I think that's worth putting into a lazy.

Reply:
Do you plan to address this here?

Reply:
Yes, will push changes shortly